### PR TITLE
#3 - Add Component calendar-videoconference-bluebar

### LINKED ISSUE #3: Integrate with the component bluebar in module esn-frontend-videoconference-calendar
Do this issue when this module is ready (now it's blocking)

### PR DESCRIPTION
**Pull request by piavgh**

### CODE REVIEW

**Review by tuanlc — APPROVED** [review]
Did you test with prod mode?

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 require('fullcalendar/dist/fullcalendar.js');
 require('fullcalendar/dist/locale-all.js');
+
```
Comment (rezk2ll) [inline src/index.js:3]:
```suggestion
```

File: src/app/app.module.js [hunk 2]
```diff
@@ -5,7 +5,9 @@ angular.module('esn.inbox-calendar', [
   'esn.notification',
   'esn.i18n',
   'esn.calendar.libs',
-  'esn.resource.libs'
+  'esn.resource.libs',
+  'linagora.esn.videoconference',
+  'linagora.esn.videoconference.calendar'
```
Comment (tuanlc) [inline src/app/app.module.js:10]: Where are these modules loaded?

Comment (piavgh) [reply src/app/app.module.js:10]: They are loaded by using this `require('esn-frontend-videoconference-calendar/src/index.js');` in `src/index.js` below

**Review by itswillta — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (piavgh) [reply src/index.js:3]: In other modules, we also have this blank line between fullcalendar and below require. So I think we should keep this convention :D

**Review by rezk2ll — APPROVED** [review]

**Comment by piavgh** [comment]
> Did you test with prod mode?

@tuanlc : I can't run prod mode with module `esn-frontend-inbox` now because it misses some dependencies because I run `npm install --ignore-scripts`

```shell
Something went wrong installing the "sharp" module

Cannot find module '../build/Release/sharp.node'

- Remove the "node_modules/sharp" directory, run "npm install" and look for errors
- Consult the installation documentation at https://sharp.pixelplumbing.com/en/stable/install/
- Search for this error at https://github.com/lovell/sharp/issues

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! esn-frontend-inbox@0.1.0 serve:prod: `webpack-dev-server --env=prod`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the esn-frontend-inbox@0.1.0 serve:prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/hp/.npm/_logs/2020-07-29T08_59_56_911Z-debug.log
```
But I guess if it can run with module `esn-frontend-calendar` properly, it can run with inbox :rofl: I will test in prod mode later

**Comment by itswillta** [comment]
> > Did you test with prod mode?
> 
> @tuanlc : I can't run prod mode with module `esn-frontend-inbox` now because it misses some dependencies because I run `npm install --ignore-scripts`
> 
> ```shell
> Something went wrong installing the "sharp" module
> 
> Cannot find module '../build/Release/sharp.node'
> 
> - Remove the "node_modules/sharp" directory, run "npm install" and look for errors
> - Consult the installation documentation at https://sharp.pixelplumbing.com/en/stable/install/
> - Search for this error at https://github.com/lovell/sharp/issues
> 
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! esn-frontend-inbox@0.1.0 serve:prod: `webpack-dev-server --env=prod`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the esn-frontend-inbox@0.1.0 serve:prod script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/hp/.npm/_logs/2020-07-29T08_59_56_911Z-debug.log
> ```
> 
> But I guess if it can run with module `esn-frontend-calendar` properly, it can run with inbox

Then I guess you can try deleting the `postinstall` script manually in `node_modules/esn-frontend-calendar` :rofl:

**Pull request merged**